app/models/teacher_project_views.rb

```diff
@@ -0,0 +1,9 @@
+class TeacherProjectViews < ActiveRecord::Base
```

Comment:
Probably this should be singular. Usually the Models are singular and the tables are plural.